hbook/h1d_test.go

```diff
@@ -294,6 +294,126 @@ func TestH1DBinsWithPanics(t *testing.T) {
 	}
 }
 
+func TestH1DAddH1D(t *testing.T) {
```

Comment:
sorry, I probably wasn't very clear (but it was late last night).
the `// Output:` thing only works with `func ExampleXXX() { ... }`.
not with `func TestXXX(t *testing.T) { ... }`.

for `TestXXX` and `hbook` histograms, I've usually resorted to running the same equivalent code with `yoda`, dumping the resulting into a `yoda` file (which is just ASCII), and then making sure (as part of the test) that `hbook` would lead to the same output `yoda` file.

I guess it's easier to just turn this `TestH1DAddH1D` into an example (say, `ExampleAddH1D()`).